include/ddsrouter/core/DDSRouter.hpp

```diff
@@ -79,8 +79,9 @@ class DDSRouter
      *
      * @return \c RETCODE_OK if configuration has been updated correctly
      * @return \c RETCODE_NO_DATA if new configuration has not changed
-     * @return \c RETCODE_BAD_PARAMETER if configuration is not well-formed
      * @return \c RETCODE_ERROR if any other error has occurred
```

Comment:
remove?

Reply:
Also, remove `@todo` and `@warning`?

Reply:
:+1: 
Yes, it is removed because now it throws an exception in error.